test/integration/mime_type_test.rb

```diff
@@ -162,6 +174,24 @@ def call(params)
     end
 
     it 'sets "Content-Type" header according to "Accept"' do
+      response = @app.get('/custom_from_accept', 'HTTP_ACCEPT' => 'application/custom')
+      response.headers['Content-Type'].must_equal 'application/custom; charset=utf-8'
+      response.body.must_equal                    'custom'
+    end
+
+    it 'sets "Content-Type" header according to "Accept"' do
+      response = @app.get('/custom_from_accept', 'HTTP_ACCEPT' => 'application/custom;q=0.9, application/json;q=0.5')
```

Comment:
I was actually wondering about this, since it seemed like the lines generally are shorter, but definitely not always.

Based on what you said, it seems like the standard is to just make sure the lines are as readable as possible... I can run through all my changes and see where they can be shortened to make it better.